externs/shaka/net.js

```diff
@@ -101,6 +101,7 @@ shaka.extern.Request;
  * @typedef {{
  *   uri: string,
  *   data: BufferSource,
+ *   status: (number|undefined)
```

Comment:
Oh! You forgot to put a comma at the end of this line. Thanks, build bot.

Reply:
Stupid me! Sorry! Definitely lack of sleep in the past days...